lib/style/single_node.ex

```diff
@@ -95,6 +95,29 @@ defmodule Styler.Style.SingleNode do
   defp style({{:., dm, [{:__aliases__, am, [:Logger]}, :warn]}, funm, args}),
     do: {{:., dm, [{:__aliases__, am, [:Logger]}, :warning]}, funm, args}
 
+  # Transform Timex defdelegates
+  defp style({{:., dm, [{:__aliases__, am, [:Timex]}, :today]}, funm, args}),
+    do: {{:., dm, [{:__aliases__, am, [:Date]}, :utc_today]}, funm, args}
+
+  defp style({{:., dm, [{:__aliases__, am, [:Timex]}, :now]}, funm, args}),
+    do: {{:., dm, [{:__aliases__, am, [:DateTime]}, :utc_now]}, funm, args}
+
+  if Version.match?(System.version(), ">= 1.15.0-dev") do
+    # Timex.{before?,after?} -> DateTime.{before?,after?}
+    defp style({{:., dm, [{:__aliases__, am, [:Timex]}, fun]}, funm, args}) when fun in [:before?, :after?],
+      do: {{:., dm, [{:__aliases__, am, [:DateTime]}, fun]}, funm, args}
+
+    # {DateTime,NaiveDateTime,Time,Date}.compare(a, b) == :lt -> {DateTime,NaiveDateTime,Time,Date}.before?(a, b)
+    defp style({:==, _, [{{:., dm, [{:__aliases__, am, [mod]}, :compare]}, funm, args}, {:__block__, _, [:lt]}]})
+         when mod in ~w[DateTime NaiveDateTime Time Date]a,
+         do: {{:., dm, [{:__aliases__, am, [mod]}, :before?]}, funm, args}
+
+    # {DateTime,NaiveDateTime,Time,Date}.compare(a, b) == :gt -> {DateTime,NaiveDateTime,Time,Date}.after?(a, b)
+    defp style({:==, _, [{{:., dm, [{:__aliases__, am, [mod]}, :compare]}, funm, args}, {:__block__, _, [:gt]}]})
+         when mod in ~w[DateTime NaiveDateTime Time Date]a,
+         do: {{:., dm, [{:__aliases__, am, [mod]}, :after?]}, funm, args}
+  end
```

Comment:
mind combining these with a `when result in `[:gt, :lt]` and then using an if to get before/after? lots of duplicate code otherwise

Reply:
Done